src/applications/vaos/services/slot/index.js

```diff
@@ -28,16 +29,32 @@ import { transformV2Slots } from './transformers';
  * @param {string} slotsRequest.clinicId clinic id
  * @param {string} slotsRequest.startDate start date to search for appointments lots formatted as YYYY-MM-DD
  * @param {string} slotsRequest.endDate end date to search for appointments lots formatted as YYYY-MM-DD
+ * @param {boolean} slotsRequest.convertToUtc check if flag to convert the start and end dates to UTC is set to true
  * @returns {Array<Slot>} A list of Slot resources
  */
-export async function getSlots({ siteId, clinicId, startDate, endDate }) {
+export async function getSlots({
+  siteId,
+  clinicId,
+  startDate,
+  endDate,
+  convertToUtc = false,
+}) {
   try {
+    // Convert to UTC ISO strings with 'Z'
+    const startUtc = convertToUtc
+      ? new Date(startDate).toISOString()
```

Comment:
nit: I think this probably doesn't matter as much here but I wonder if we would prefer `toISOString` or `format(startDate, DATE_FORMATS. ISODateTimeUTC)`. @vbahinwillit what do you think since you've been working with date-fns a lot more.